src/features/expenses/AddExpense/MainInfo.styled.ts

```diff
@@ -1,4 +1,5 @@
-import { TextRegular } from "components";
+import { TextRegular, TextUnderline } from "components";
+import { ButtonTransparent } from "components/Button/index.styled";
```

Comment:
I believe you forgot to register this button in general component import, that’s why you import comes from button. Please have a look how it’s done for the rest of texts and buttons and etc